pkg/kubelet/kubelet.go

```diff
@@ -589,6 +592,45 @@ type Kubelet struct {
 	daemonEndpoints *api.NodeDaemonEndpoints
 }
 
+// KubeletAuth implements AuthInterface
```

Comment:
suggestion: moving this new block of code outside of kubelet.go, perhaps to a new file.